frontend/src/Pages/EventAttendance.tsx

```diff
@@ -185,8 +188,16 @@ export default function EventAttendance() {
             return newMods;
         });
     }
-
     async function onSubmit() {
+        const canEditAttendance =
+            thisSelectedClassStatus !== SelectedClassStatus.Completed &&
+            thisSelectedClassStatus !== SelectedClassStatus.Cancelled;
+        if (!canEditAttendance) {
+            console.error(
+                'Cannot update attendance for completed or canceled classes'
+            );
```

Comment:
@corypride We shouldn't be keeping console in here. This should be an error visible to the user.